app/src/main/res/values/subtitle_colors.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Define the colors in code instead of XML. I think an enum would be a good solution for this.